ginoauth2.go

```diff
@@ -219,9 +219,11 @@ func getTokenContainerForToken(o Options, token *oauth2.Token) (*TokenContainer,
 		errorf("[Gin-OAuth] JSON.Unmarshal failed caused by: %s", err)
 		return nil, err
 	}
-	if _, ok := data["error_description"]; ok {
-		var s string
-		s = data["error_description"].(string)
+	if si, ok := data["error_description"]; ok {
+		s, ok := si.(string)
+		if !ok {
+			s = ""
```

Comment:
Do you expect this line to be executed? If yes, why is it okay to ignore `data["error_description"]`?